src/ucp/core/ucp_context.c

```diff
@@ -444,6 +444,12 @@ static ucs_config_field_t ucp_context_config_table[] = {
    "directory.",
    ucs_offsetof(ucp_context_config_t, proto_info_dir), UCS_CONFIG_TYPE_STRING},
 
+  {"REG_NONBLOCK_MEM_TYPES", "",
```

Comment:
general question, are we going to warn/diag if user sets this value to some memory type and we do not have any MD which supports non-blocking registration for such type?